anycloud/docs/how-to/aws-postgres.md

```diff
@@ -0,0 +1,132 @@
+
+4) Take the `accessKeyId` and `secretAccessKey` from step 2 and create AWS `Credentials` in AnyCloud. You will need to pick a name or alias for the `Credentials`. The initial value will be `aws`. In this example, we will call it `mystartup-aws`.
+
+```bash
+$ anycloud credentials new
+? Pick cloud provider for the new credentials ›
+❯ AWS
+  GCP
+  Azure
+Name for new Credentials: mystartup-aws
+AWS Access Key ID: ******************
+AWS Secret Access Key: ******************
+Successfully created "mystartup-aws" Credentials
+```
+
+## Deploy the REST API using AnyCloud and link it to the PostgreSQL instance
+
+1) First download Prisma's example and install dependencies
+
+```
+curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/typescript/rest-express
+mv rest-express anycloud-node-aws-psql
+```
+
+Install npm dependencies:
+
+```
+cd anycloud-node-aws-psql
+npm install
+```
+
+2) Add `npm scripts` to the `package.json` file
+
+```
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf dist",
+    "start": "node ./dist/src/index.js",
+  }
+```
+
+3) Define the `Dockerfile`: 
+
+```bash
+FROM node:lts
+
+COPY . .
+
+RUN npm install
+RUN npm run build
+CMD npm run start
+```
+
+4) Initialize a `git` repository
+
+```bash
+git init
+git add -A
+git commit -m "Initial commit"
+```
+
+5) Use the AnyCloud CLI to create an `anycloud.json` file in the project directory and define a `Deploy Config`. You will need to pick a name or alias for the `Deploy Config`. The initial value will be `staging`. Since there is only `Credentials`, the CLI will default to that for your new `Deploy Config`.
+
+```bash
+$ anycloud config new
+? Name for new Deploy Config › staging
+? Pick Credentials to use ›
+❯ mystartup-aws
+  Create new Credentials
+? Region name › us-east-1
+? Virtual Machine Type › m5.large
+? Do you want to add another region to this Deploy Config? › n
+Successfully created "staging" Credentials
+```
+
+6) Deploy the Node.js server to your AWS account using the AnyCloud CLI. The only argument required is the value of one of the keys in `anycloud.json` to reference a deploy configuration. We use `staging` which we previously defined:
+
+```bash
+$ anycloud new
+? Pick Deploy Config for App ›
+❯ staging
+? Optional App name ›
+▇ Creating new App
+```
+
+7) Now go to [AWS RDS](https://console.aws.amazon.com/rds/home) and select the PostgreSQL instance in order to include the [Security Groups](https://console.aws.amazon.com/ec2/v2/home?#SecurityGroups) for the newly created AnyCloud app. It will have the format `anycloud-${app-id}-${uuid}`. First click `Modify`
```

Comment:
Since the users might not know the `app-id`, do you think it would be necessary to add an extra step checking `anycloud list`?